assets/css/style.css

```diff
@@ -0,0 +1,77 @@
+.module, body .wrapper .all-added-nums-sum .result, body .wrapper .all-added-nums-sum .added-nums-container .current-added-elem {
+  color: #625F5F;
```

Comment:
The products and map file of the sass should not be committed. 
You can use .gitignore for this